lib/std/Thread.zig

```diff
@@ -584,7 +584,7 @@ const PosixThreadImpl = struct {
                 };
                 return @intCast(usize, count);
             },
-            .solaris => {
+            .solaris, .serenity => {
```

Comment:
FWIW: I did the Solaris port, and this comment refers to a unique device. I think modifying the comment to add "On Solaris ...", or creating a serenity-specific branch in the case would be better.